types/node-schedule/index.d.ts

```diff
@@ -8,7 +8,7 @@
 import { EventEmitter } from 'events';
 
 /** The callback executed by a Job */
-export type JobCallback = () => void;
+export type JobCallback = (fireDate?: Date) => void;
```

Comment:
a callback should not have an optional parameter. 
the caller of this callback will either call it with `fireDate` or not..

Reply:
also `JobCallback|(() => void)` is meaningless. since a function with no parameters is assignable to one with more.. e.g.:

```ts
function f( callback: (a: number) => void ) { ... }

f( (a) => {} ); // OK
f( ()=> {} );  // OK too.

```